src/app/components/about/about.component.html

```diff
@@ -1,22 +1,31 @@
 <app-header-static></app-header-static>
 
-<div class="about-content container-top about-container">
-	<img src="../assets/images/about_image.png" />
```

Comment:
It looks like you didn't remove this image from assets folder. Is it needed somewhere else?

Reply:
oh, no I don't thing so, I'll remove it now :)

Reply:
@Shekharrajak done